_posts/2019-01-30-custom-safetyGraphics-workflows.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: Custom safetyGraphics Workflows
+author: Jeremy Wildfire
+---
+
+The `safetyGraphics` R package now supports for custom workflows allowing users to preload their own charts and data sets in the safetyGraphics Shiny Application. The best way to learn more is the new [Custom Workflows vignette](https://github.com/SafetyGraphics/safetyGraphics/wiki/Vignette:-Custom-Workflows) - which has been briefly summarized below.
```

Comment:
suggest linking to CRAN instead - https://cran.r-project.org/web/packages/safetyGraphics/vignettes/customWorkflows.html